### PR TITLE
add s390x owners to sig-buildsystem-reviewers.

### PR DESCRIPTION
**Pull request by vamsikrishna-siddu**
### What this PR does
Before this PR:
There were no s390x sig-buildsystem-reviewers in kubevirt.
After this PR:
There will be s390x sig-buildsystem-reviewers in kubevirt.

Fixes #

### Why we need it and why it was done in this way
We are planning to add s390x support for the  kubevirt builder image. To ensure the kubevirt to built successfully on s390x. I am adding myself to the sig-buildsystem-reviewers. So that i will be notified about PRs with sig-buildsytem label and helps me review those PRs.
Kubevirt builder image PR link: https://github.com/kubevirt/kubevirt/pull/11097
/cc @brianmcarey  @dhiller 
The following tradeoffs were made:

The following alternatives were considered:

Links to places where the discussion took place: 
https://github.com/kubevirt/kubevirt/pull/11097#issuecomment-1916646807
https://github.com/kubevirt/kubevirt/pull/11097#issuecomment-2022786437

### Checklist

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

### Release note

```release-note
      NONE
```

### CODE REVIEW

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -139,6 +139,8 @@ aliases:
   sig-buildsystem-reviewers:
     - brianmcarey
     - enp0s3
+  # Owns s390x, and ensures that kubevirt can be built on s390x.
```
Comment (dankenigsberg) [inline OWNERS_ALIASES:142]: this makes it seem that xpivarc is covered by this comment

Comment (vamsikrishna-siddu) [reply OWNERS_ALIASES:142]: Hi @dankenigsberg  i can move my name below xpivarc but this will disrupt alphabetical order.

Comment (vamsikrishna-siddu) [reply OWNERS_ALIASES:142]: can i put something like this?
```
 # Owns s390x, and ensures that kubevirt can be built on s390x.
    - vamsikrishna-siddu
 # end of s390x owners
    - xpivarc
```

Comment (vamsikrishna-siddu) [reply OWNERS_ALIASES]: Hi @dankenigsberg  i have removed the comment and pushed the latest changes. Can you please review it again. Thanks

**Review by dankenigsberg — APPROVED** [review]

**Comment by vamsikrishna-siddu** [comment]
@iholder101  Can you please review this PR?

**Comment by iholder101** [comment]
> @iholder101 Can you please review this PR?

Hey @vamsikrishna-siddu!
First of all, thanks a lot for your contributions!

TBH, at a first glance I would think that someone would need a fair amount of contributions and reviews in order to become a reviewer. I am open to changing my mind on this if there's a good enough reason.

/cc @fabiand @vladikr @dhiller @brianmcarey @xpivarc 

I would start with asking - why is it necessary?
Even if @vamsikrishna-siddu is the owner of s390x support, how does it help if he becomes a reviewer? Let me remind that becoming a reviewer only means that he would get notified for PRs. Personally I don't really understand why it is necessary here. I also remind that even without becoming a reviewer every community member can `/lgtm`.

So, I think the alternative is that when a PR related to s390x is introduced, someone from SIG-API (that is aware of this) would `/cc` @vamsikrishna-siddu for a review. SIG-API members can even add a comment similar to "PR looks good at a first glance, howevr @vamsikrishna-siddu is the expert on this. WDYT?".

If there is a concrete reason however, I think that as @fabiand [said](https://github.com/kubevirt/kubevirt/pull/11097#issuecomment-1916646807), we should add a comment about owning only s390x related stuff.

**Comment by iholder101** [comment]
/lgtm cancel

I thought that doing so in a `markdown syntax` won't actually trigger the bot :)

**Comment by dhiller** [comment]
> > @iholder101 Can you please review this PR?
> 
> Hey @vamsikrishna-siddu! First of all, thanks a lot for your contributions!
> 
> TBH, at a first glance I would think that someone would need a fair amount of contributions and reviews in order to become a reviewer. I am open to changing my mind on this if there's a good enough reason.
> 
> /cc @fabiand @vladikr @dhiller @brianmcarey @xpivarc
> 
> I would start with asking - why is it necessary? Even if @vamsikrishna-siddu is the owner of s390x support, how does it help if he becomes a reviewer? Let me remind that becoming a reviewer only means that he would get notified for PRs. Personally I don't really understand why it is necessary here. I also remind that even without becoming a reviewer every community member can `/lgtm`.
> 
> So, I think the alternative is that when a PR related to s390x is introduced, someone from SIG-API (that is aware of this) would `/cc` @vamsikrishna-siddu for a review. SIG-API members can even add a comment similar to "PR looks good at a first glance, howevr @vamsikrishna-siddu is the expert on this. WDYT?".
> 
> If there is a concrete reason however, I think that as @fabiand [said](https://github.com/kubevirt/kubevirt/pull/11097#issuecomment-1916646807), we should add a comment about owning only s390x related stuff.

Hey @iholder101 I get your concerns about having a contribution record. However, this addition is not about a record but a commitment to ownership. We are asking for @vamsikrishna-siddu (and @jschintag for that matter also) to own their contributions wrt s390x arch, and making that explicit by adding themselves to the `sig-buildsystem-reviewers`. This is due to the lack of not having a better group in place, all owned to the amount of people we have in community currently.

See also https://github.com/kubevirt/kubevirt/pull/11097#issuecomment-2022786437, hope that brings more clarity.

File: OWNERS_ALIASES [hunk 2]
```diff
@@ -139,6 +139,7 @@ aliases:
   sig-buildsystem-reviewers:
     - brianmcarey
     - enp0s3
+    - vamsikrishna-siddu
```
Comment (dhiller) [inline OWNERS_ALIASES:142]:
```suggestion
    - vamsikrishna-siddu   # s390x arch
```

Comment (vamsikrishna-siddu) [reply OWNERS_ALIASES]: Hi @dhiller  I have updated the OWNER_ALIASES with the suggested changes. Can you please review it once.?

**Comment by iholder101** [comment]
> However, this addition is not about a record but a commitment to ownership.

> This is due to the lack of not having a better group in place

Thanks @dhiller.
I understand your reasoning, just wonder if there's a better way to express this ownership.

I, of course, thank @vamsikrishna-siddu and @jschintag a lot for their commitment and intentions and appreciate it a lot. On the other hand, they have near to zero contributions to Kubevirt [1][2]. Making them reviewers of a SIG, because they have the **intention** of future development without any actual contributions so far just seems wrong to me.

I think that we either need to wait until they contribute code and become a significant part of `sig-buildsystem`, or think of a better way to express that they're the owner of this effort.

In Kubernetes, for example, there's the concept of an "area" (e.g. `area/kubelet`), which is a component ownership rather than a SIG ownership. Perhaps we can explore this direction as well.

[1] https://github.com/kubevirt/kubevirt/pulls?q=is%3Apr+is%3Amerged+author%3Avamsikrishna-siddu
[2] https://github.com/kubevirt/kubevirt/pulls?q=is%3Apr+is%3Amerged+author%3Ajschintag

**Comment by iholder101** [comment]
BTW, using areas can also help us be more fine-grained. Especially in a situation when some of the sigs (e.g. compute) covers a large portion of the repo's code. So going this way might actually benefit our current SIG efforts in general.

Some "areas" that come into mind:
* node-labeller
* virtctl
* VM cloning

and more.
FYI @jean-edouard

**Comment by dhiller** [comment]
> > However, this addition is not about a record but a commitment to ownership.
> 
> > This is due to the lack of not having a better group in place
> 
> Thanks @dhiller. I understand your reasoning, just wonder if there's a better way to express this ownership.
> 
> I, of course, thank @vamsikrishna-siddu and @jschintag a lot for their commitment and intentions and appreciate it a lot. On the other hand, they have near to zero contributions to Kubevirt [1][2]. Making them reviewers of a SIG, because they have the **intention** of future development without any actual contributions so far just seems wrong to me.
> 
> I think that we either need to wait until they contribute code and become a significant part of `sig-buildsystem`, or think of a better way to express that they're the owner of this effort.
> 
> In Kubernetes, for example, there's the concept of an "area" (e.g. `area/kubelet`), which is a component ownership rather than a SIG ownership. Perhaps we can explore this direction as well.
> 
> [1] [kubevirt/kubevirt/pulls (is:merged author:vamsikrishna-siddu)](https://github.com/kubevirt/kubevirt/pulls?q=is%3Apr+is%3Amerged+author%3Avamsikrishna-siddu+sort%3Aupdated-desc) [2] [kubevirt/kubevirt/pulls (is:merged author:jschintag)](https://github.com/kubevirt/kubevirt/pulls?q=is%3Apr+is%3Amerged+author%3Ajschintag+sort%3Aupdated-desc)

These are all good points @iholder101 , thank you for that! I admittedly struggled already to find a better way.

We were discussing this in-person and thought about using working groups, but those are timely groups, so maybe `area/s390x` (as well as `area/arm` for that matter) might be a better way to go.

Summoning @fabiand on that matter, WDYT?

**Pull request closed without merging**